client/components/formatted-header/docs/example.jsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FormattedHeader from '..';
+
+export default function FormattedHeaderExample() {
```

Comment:
Yes, good call about the `displayName`. But you don't need to use a class - you can have it defined as a stateless functional component and still assign a `displayName` to it - see the `Banner` example for reference: https://github.com/Automattic/wp-calypso/blob/master/client/components/banner/docs/example.jsx